site/src/components/page-faq/question-section.scss

```diff
@@ -13,13 +13,32 @@
     align-self: center;
 }
 
+section { 
+    max-height: 0; 
+    overflow: hidden; 
+    -webkit-transition: max-height 0.25s ease-in-out;
+    -moz-transition: max-height 0.25s ease-in-out;
+    -o-transition: max-height 0.25s ease-in-out;
+    transition: max-height 0.25s ease-in-out;
```

Comment:
The SASS parser will take care of adding platform specifics if needed

Reply:
nice thanks!